lib/jekyll/post.rb

```diff
@@ -118,6 +119,14 @@ def title
       data.fetch('title') { titleized_slug }
     end
 
+    # Public: the Post excerpt_separator, from the YAML Front-Matter or site default
+    #         excerpt_separator value
+    #
+    # Returns the post excerpt_separator
+    def excerpt_separator
+      data.fetch('excerpt_separator') { site.config['excerpt_separator'].to_s }
```

Comment:
@alfredxing the form I've used just the same as the other fetchers. but I'd prefer the view: `data['excerpt_separator] || site.config['excerpt_separator'].to_s`